src/rtc/h5_prep.py

```diff
@@ -411,3 +499,37 @@ def save_hdf5_dataset(ds_filename, h5py_obj, root_path,
 
     del gdal_ds
 
+
+def populate_rfi_info(h5py_obj, burst, rfi_root_path):
```

Comment:
I thought adding RFI placeholders into PR would make sense, also allow us to save some time to review because most of the changes in this PR is about populating metadata.